src/app/tracing/dialog/dialog-import-warning-modal/dialog-import-warning-modal.component.html

```diff
@@ -0,0 +1,15 @@
+<fcl-dialog-movable>
+    <div class="fcl-dialog-import-warning-modal" mat-dialog-content>
```

Comment:
`class="fcl-dialog-import-warning-modal"`can be obmitted

Reply:
- [ ] remove class